recipes/gtk4/build.sh

```diff
@@ -0,0 +1,45 @@
+#! /bin/bash
+
+set -ex
+
+export PKG_CONFIG_PATH=$PKG_CONFIG_PATH:$BUILD_PREFIX/lib/pkgconfig
+
+export XDG_DATA_DIRS=${XDG_DATA_DIRS}:$PREFIX/share
+
+meson_config_args=(
+    -Dgtk_doc=false
+    -Dman-pages=false
+    -Dintrospection=enabled
+    -Dbuild-examples=false
+    -Dbuild-tests=false
+    -Dwayland-backend=false
+)
+
+if test $(uname) == 'Darwin' ; then
```

Comment:
It doesn't matter for now as we don't cross-compile between OSs but it's slightly better to use something like `if [[ "${target_platform}" == "osx-"* ]]; then` (and the same for Linux slightly lower down)